app/src/main/java/com/lab/skuld/ui/theme/Theme.kt

```diff
@@ -47,15 +41,18 @@ private val LightColorPalette = lightColors(
 
 @Composable
 fun SkuldFrontendTheme(
+
     darkTheme: Boolean = isSystemInDarkTheme(),
```

Comment:
Hmm, but then we aren't testing for system theme. It would be nice to have a "System theme" option that does what we were doing previously

Reply:
Added(and tested) new default option "System theme" to settings.